gitpod-io/gitpod
# [ws-daemon] Support rootfs quota

## Description
This PR adds XFS-based quota support to the rootfs of containers.
Ephemeral storage in Kubernetes does not encompass this.

## How to test
- Run on a cluster with XFS on the container root FS
- Set the `containerRootfsQuota` to a value other than zero in the config
- Start a workspace and look for errors from ws-daemon
- Check the `df -h` output in the workspace

## Release Notes

```release-note
NONE
```
